README.md

```diff
@@ -40,6 +40,9 @@ PRs accepted.
 
 Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
 
+## Copyrights
```

Comment:
This is good thanks ! 
But given that the other changes are less obvious perhaps it would be best to seperate this into another PR please :)